src/frontend-admin/src/components/SoftwareCard/SoftwareCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react";
+
+// Components
+import { Card } from "../reusables/Card/Card";
+import {
+  DashboardTable,
+  IDashboardTableDatum
+} from "../reusables/DashboardTable/DashboardTable";
+import { AxiosService } from "../../services/AxiosService/AxiosService";
+
+// Styles
+import styles from "./SoftwareCard.module.css";
+
+// Primary Component
+export const SoftwareCard: React.SFC<{ className?: string }> = props => {
+  const { className } = props;
+  let initData: IDashboardTableDatum[] = [];
+  const [tableData, setTableData] = useState(initData);
+
+  const axios = new AxiosService("accessToken", "refreshToken");
+  useEffect(() => {
+    //TODO: find out endpoint name
+    axios.get("/Software", setTableData);
+  }, [setTableData]);
```

Comment:
The dashboardTable.tsx should have the axios stuff in the actual component. To be able to use it on the department card with the dropdown, we could pass through the endpoint as a prop